randovania/gui/widgets/changelog_widget.py

```diff
@@ -22,9 +36,22 @@ def __init__(self, all_change_logs: dict[str, str]):
             label = DelayedTextLabel()
             label.setAlignment(QtCore.Qt.AlignmentFlag.AlignTop)
             label.setObjectName(f"label {version_name}")
-            label.setTextFormat(QtCore.Qt.MarkdownText)
+            label.setOpenExternalLinks(True)
+            label.setTextFormat(QtCore.Qt.TextFormat.MarkdownText)
             label.setText(version_text)
             label.setWordWrap(True)
 
             scroll_area.setWidget(label)
-            self.addTab(scroll_area, version_name)
+            self.changelog.addWidget(scroll_area)
+
+            self.select_version.addItem(version_name)
+
+        self.changelog.setCurrentIndex(0)
+
+    def select_version_index_changed(self) -> None:
+        # Used "Any" type hint to bypass mypy raising "expecting QWidget"
+        selected_widget: Any = self.findChild(
```

Comment:
Why wouldn't QWidget as a type hint work here if it's whats expected? Does findChild not return that?